README.md

```diff
@@ -1,2 +1,15 @@
 # codex.docs.chart
 Helm chart for deploying CodeX Docs
+
+## Running
+
+Execute the following commands from the repository root folder:
+```
+kubectl create ns codex-docs
+helm install codex-docs ./ -n codex-docs
```

Comment:
We can omit the `kubectl create ns` command and call helm with the `--create-namespace option`:

`helm install codex-docs ./ -n codex-docs --create-namespace`